src/harness/reference_models/dpa/dpa_mgr_example.py

```diff
@@ -136,3 +136,17 @@
                                     do_abs_check_single_uut=True)
   print 'Move list output: ' + str(dpa_uut.GetMoveListMask(channel))
   print 'Check Interference @%.2fdB margin: %s' % (margin_db, 'OK' if check else 'FAIL')
+
+  # Simulate the BuildDpa feature
+  print '-- Test Build DPAs --'
+  dpa_pt = dpa_mgr.BuildDpa('Pensacola')
+  print 'Pensacola:'
+
+  print dpa_pt
+  dpa_zone = dpa_mgr.BuildDpa('East3',
+                              protection_points_method='default(4,2,1,1)')
```

Comment:
I was expecting to get 8 protection points (4 on the front border, 2 on the back border, 1 inside the front zone, and 1 inside the back zone). However, the results show only 7 protection points (see below). Did I do anything wrong?
Dpa(protected_points=[ProtectionPoint(latitude=30.2777761712794, longitude=-86.8723702411151), ProtectionPoint(latitude=28.936338148926364, longitude=-89.25247288586112), ProtectionPoint(latitude=30.13606756959702, longitude=-88.64110991511458), ProtectionPoint(latitude=29.80435257686979, longitude=-86.89639795810538), ProtectionPoint(latitude=28.776692879972796, longitude=-89.07071605705865), ProtectionPoint(latitude=30.14701158854293, longitude=-87.70039734848852), ProtectionPoint(latitude=28.450159209933283, longitude=-87.53895141517933)], ...)

Reply:
2 things:
 - the line sampling routine was missing the last end of the line - I fixed, so probably the reason for the missing point.
 - but in general this routine does not insure perfect number is output as requested, especially for the zone part. Look at it: we take the area then deduce the step, then grid the polygon. This is quite approximate, but sufficient for our purpose.